examples_tests/13.BRDF_Explorer/main.cpp

```diff
@@ -0,0 +1,99 @@
+#define _IRR_STATIC_LIB_
+
+/* GCC compile Flags
```

Comment:
remove this comment, also from other examples if you desire to do so

Reply:
Tried removing other examples, my editor produces ^M as EOF so there're more changes in a commit. I'm only committing for my example for now.